lib/iiif_print/data/fileset_helper_decorator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# OVERRIDE IIIF Print v1.0.0 to deal with Attachment ids
+
+IiifPrint::Data::FilesetHelper.module_eval do
+  def first_fileset
```

Comment:
Should this change be done in iiif print now that we discovered a similar issue in ADL?

Reply:
Yes, you're right.  Also Jeremy did say it's safe to update IIIF Print since the Rodeo stuff isn't enabled, I'll make a PR for that.